draft-ietf-tls-esni.md

```diff
@@ -520,9 +524,11 @@ to be padded using TLS record layer padding.
 ## Handling the server response {#handle-server-response}
 
 As described in {{server-behavior}}, the server MAY either accept ECH and use
-ClientHelloInner or reject it and use ClientHelloOuter. However, there is no
-indication in ServerHello of which one the server has done and the client must
-therefore use trial decryption in order to determine this.
+ClientHelloInner or reject it and use ClientHelloOuter. In handling the server's
+response, the client's first step is to determine which was used. To do so, it
```

Comment:
Nit: s/which/which ClientHello/

Reply:
Did s/which/which value/. I don't think we should be calling ClientHelloInner a "ClientHello", since ClientHelloInner isn't a handshake message on the wire.